### ISSUE TITLE
Improve app look, behavior and accessibility with non-standard interface sizes

### ISSUE THREAD

**Issue by PatrykMis** [accrescent/accrescent#129]
#### Steps to reproduce:

Settings -> display -> Interface Size (and additionally Font Size) set to big/large/very large (just highest possible values).

#### Observed behavior

The app looks quite strange then (I am visually impaired user so can't describe it in detail), but for me, more important is that "open" button of second app on the list is not announced at all by TalkBack. With standard (default) font and interface size, the label of "open" button is announced correctly.

Don't know it's issue of Accrescent, of some composable or of Compose itself - I just learn coding and I'm total beginner.

**Comment by lberrymage** [comment]
I think the best solution to this is something I've suggested before - moving the install/uninstall/open buttons out of the app lists and into the app detail screen. A similar issue occurs when the app label (display name) is too long, which happens on certain form factors even in English, not to mention other languages which often have longer strings for the button texts.

**Comment by PatrykMis** [comment]
Great solution which works in other apps (unfortunately not much of them) and which I'm thinking about is this: <https://developer.android.com/jetpack/compose/accessibility#custom-actions>. We then can define Open and Uninstall as custom actions. However, it obviously does not solve display issues.
